zerver/openapi/zulip.yaml

```diff
@@ -24311,10 +24315,31 @@ components:
         last_edit_timestamp:
           type: integer
           description: |
-            The UNIX timestamp for when the message was last edited,
-            in UTC seconds.
+            The UNIX timestamp for when the message's content was last edited, in UTC seconds.
+
+            Not present if the message's content has never been edited.
+
+            Clients are recommended to use this field, rather than parsing the
+            `edit_history` object, to display EDITED indicators on messages that have been
```

Comment:
small error:
```suggestion
            `edit_history` array, to display EDITED indicators on messages that have been
```

(In JS, Array is a subtype of Object, so "object" would be technically not incorrect, even though confusing. But in JSON, arrays and objects are unrelated types.)